data_migration.go

```diff
@@ -0,0 +1,53 @@
+package influxdb
```

Comment:
I'm concern that this pattern isn't going to be something that works long term. I imagine that this will likely be something we repeat with some regularity.